internal/witness/cmd/witness/internal/witness/witness_test.go

```diff
@@ -228,15 +240,29 @@ func TestGetSTH(t *testing.T) {
 				}
 			}
 			// Try to get the latest STH.
-			_, err := w.GetSTH(test.queryID)
+			sthRaw, err := w.GetSTH(test.queryID)
 			if !test.wantThere && err == nil {
 				t.Fatalf("returned an STH but shouldn't have")
 			}
 			// Check to see if we got something.
-			// TODO(meiklejohn): check witness signature once those
-			// have been implemented.
 			if test.wantThere && err != nil {
 				t.Fatalf("failed to get latest: %v", err)
+				sv, err := ct.NewSignatureVerifier(wPK)
```

Comment:
Is this code in the right place?
Not sure if the diff is wonky, but it looks like we'll hit the `t.Fatalf()` and that'll be the end of the test?

Reply:
Oh yeah, strange!  This should be fixed now.